// [[Reason: Semantically invalid. TSC does not emit .d.ts]] ////

//// [tests/cases/compiler/declarationEmitInvalidReferenceAllowJs.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,7 +1,8 @@
 
 
 //// [declarationEmitInvalidReferenceAllowJs.d.ts]
+/// <reference path="invalid.d.ts" />
 declare var x: number;
 
 /// [Errors] ////
 
